default_config.yaml

```diff
@@ -129,7 +129,12 @@ measuring:
     negatives: 0.3
     bad pixels: 1
     offsets: 5.0
-    filter bank: 1.0
+    filter bank: 1
+  deletion_thresholds:
```

Comment:
I could also set the deletion thresholds to None to mimic current behavior instead of an identical dict like this